spec/factories/trainees.rb

```diff
@@ -221,7 +221,7 @@
 
     trait :trn_received do
       submitted_for_trn
-      trn { SecureRandom.uuid }
+      trn { Faker::Number.number(digits: 9) }
```

Comment:
```suggestion
trn { Faker::Number.number(digits: 7) }
```

Isn't it a 7 digit number? @gpeng could you confirm?

Reply:
I think it is 7, I've changed!